CopyKeys.ps1

```diff
@@ -176,11 +177,12 @@ function Get-VirtualMachines
         $LocationDropDown.Enabled = $true
         $LocationDropDown.Text = ""
 
-        $VmList = (Get-AzureRmVm -ResourceGroupName $ResourceGroupName) | sort Name
+        $VmList = (Get-AzVm -ResourceGroupName $ResourceGroupName) | sort Name
 
         foreach ($Item in $VmList)
         {
-            if ($Item.StorageProfile.OsDisk.EncryptionSettings.Enabled -eq "True")
+            #if ($Item.StorageProfile.OsDisk.EncryptionSettings.Enabled -eq "True")
+            if ((Get-AzVmDiskEncryptionStatus -ResourceGroupName $ResourceGroupName -VMName $item.Name).OsVolumeEncrypted -eq "Encrypted")

```

Comment:
Please revert this line change.  We don't want to make an extra call during the UI as it increases the load time.
Also, some VMs won't show up. These are the single pass encrypted VMs which are not supported by this script as of today.
So, only 2-pass encrypted VMs should be shown. 
And even if you want to see both 1 pass and 2 pass encrypted VMs, I would suggest the following

`if (($Item.Extensions -ne $null) -and 
($Item.Extensions.Id | % {$_.split('/')[-1].tolower().contains('azurediskencryption')}) -contains $true)`